lib/components/form/form.css

```diff
@@ -414,3 +414,26 @@
 .otp .date-time-modal .date-time-selector.with-desktop-touchscreen input::-webkit-datetime-edit {
   margin-inline-end: 8px;
 }
+
+.batch-routing-panel-location-fields {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 40px 10px;
```

Comment:
The grid stuff that follows is not needed if you do for this CSS class:
```suggestion
  display: block;
  position: relative;
```

Note that the switch button container's existing CSS should handle the positioning correctly already with this suggestion.

Reply:
<img width="438" alt="Screenshot 2023-02-08 at 1 21 18 PM" src="https://user-images.githubusercontent.com/86619099/217618107-93a9dea8-ed79-4a7b-934f-0c1aba433ade.png">
Not sure if I missed something but this doesn't exactly work

Reply:
For the switch button container, apply `position: absolute; right: ..., top: ...` and for its parent element, apply `position: relative`.

Reply:
I tried this and couldn't get it to work, since the container moves around so much (and forget about mobile). Maybe you'll have more luck